vizro-core/tests/unit/vizro/models/_components/form/test_switch.py

```diff
@@ -0,0 +1,104 @@
+"""Unit tests for Switch."""
```

Comment:
Tests in `test_filter.py` need updating too:
* a new `df_boolean = pd.DataFrame({"shared_column": [False]})` like with other types
* `TestFilterFunctions` should test new boolean columns of both `False/True` and `0/1` (don't worry about mixed cases)
* `test_disallowed_selectors_per_column_type` 
* probably other stuff too, I didn't check through the whole file 😅 So please do so!